docs/_posts/2019-07-14-Magic-Mirror-Project-Introduction-and-Kick-Off-Notification.md

```diff
@@ -0,0 +1,110 @@
+
+---
+title: Magic Mirror Project Introduction and Kick Off Notification
+date: 2019-07-14 20:00 +0800
+author: newip
+comments: true
+mathjax: true
```

Comment:
The `mathjax` I think is not necessary for this post, because it did not contain LaTex.

It should be safe to delete this line.

Reply:
Done。